### PR TITLE
Update data_manager_primer_scheme_bedfiles

### PR DESCRIPTION
**Pull request by pvanheus**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by pvanheus** [comment]
This update addresses 3 things:
1. Sanitizing BED files to make sure they are valid BED
2. Making tests work
3. Adding help test and a citation to the PrimalSeq / iVar paper

File: data_managers/data_manager_primer_scheme_bedfiles/data_manager/install_primer_scheme_bedfiles.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<tool id="data_manager_primer_scheme_bedfiles" name="BED-format primer scheme data manager" version="0.0.10" tool_type="manage_data" profile="19.05">
+<tool id="data_manager_primer_scheme_bedfiles" name="BED-format primer scheme data manager" version="0.0.11" tool_type="manage_data" profile="19.05">
     <requirements>
         <requirement type="package">python</requirement>
         <requirement type="package" version="2.22.0">requests</requirement>
```
Comment (nsoranzo) [inline data_managers/data_manager_primer_scheme_bedfiles/data_manager/install_primer_scheme_bedfiles.xml:4]:
```suggestion
        <requirement type="package" version="2.24.0">requests</requirement>
```

File: data_managers/data_manager_primer_scheme_bedfiles/data_manager/install_primer_scheme_bedfiles.py [hunk 2]
```diff
@@ -48,7 +67,7 @@ def install_primer_file(
 ):
     name = re.sub(r"\W", "", str(primer_name).replace(" ", "_"))
     output_filename = os.path.join(output_directory, name + ".bed")
-    shutil.copyfile(input_filename, output_filename)
+    write_good_bed(open(input_filename), output_filename)
```
Comment (nsoranzo) [inline data_managers/data_manager_primer_scheme_bedfiles/data_manager/install_primer_scheme_bedfiles.py:70]: I'd open both files inside `write_good_bed()`.

Comment (nsoranzo) [inline data_managers/data_manager_primer_scheme_bedfiles/data_manager/install_primer_scheme_bedfiles.py:70]:
```suggestion
    with open(input_filename) as fh:
        write_good_bed(fh, output_filename)
```

**Pull request merged**